prime-router/settings/schemas/receiver.json

```diff
@@ -0,0 +1,447 @@
+{
+  "$ref": "#/definitions/Receiver",
+  "definitions": {
+    "Receiver": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "organizationName": {
+          "type": "string"
+        },
+        "topic": {
+          "$ref": "#/definitions/Topic"
+        },
+        "customerStatus": {
+          "$ref": "#/definitions/CustomerStatus"
+        },
+        "translation": {
+          "anyOf": [
+            {
+              "$ref": "#/definitions/Translation"
+            },
+            {
+              "type": "null"
+            }
+          ]
+        },
+        "jurisdictionalFilter": {
```

Comment:
It might be nice to put filters in their own section like it is done at the higher level.

Reply:
Looks like this still needs to happen?

Reply:
will check